Jenkinsfile_CNP

```diff
@@ -48,24 +48,29 @@ def vaultOverrides = [
     'spreview': 'saat'
 ]
 
-// vars needed for functional tests
+// Env variables to switch pipeline logic and wiring per type of build
+env.DATA_STORE_DEVELOP_PR       = "PR-601" // This doesn't change frequently, but when it does, only change this value.
```

Comment:
I think you could use a normal variable here? i.e.
def DATA_STORE_DEVELOP_PR = "PR-601"
see
https://github.com/hmcts/ccd-data-store-api/blob/cbdbb95e62ff2fdf9acfa695fd4710ab51bfa81d/Jenkinsfile_CNP#L14

Reply:
Sure, good point.